### ISSUE TITLE
Please make terminal trains distinct visually

### ISSUE THREAD

**Issue by egoine** [jamespfennell/react-sandbox#37]
Let me explain with an example:

When I take a quick look at eg whitehall station, brooklyn direction, I see on top of the lost the next train in 3 mins, for example. 

But actually this is just the W train for which Whitehall is the terminal session. So it is easy to get confused and think the next train is in 3 mins, when in fact the next train is the second listed for brooklyn, eg the R in 7 mins. 

Not a huge deal but I constantly get it wrong 🙂

I’m not sure what UX design would be best but it would help if somehow this gotcha was easy(er) to spot visually. 

Apart from that, fantastic app! Thanks!!

**Comment by jamespfennell** [comment]
Yeah this does sound confusing! I think there are two options, both easy to implement:

1. Filter out trains that are terminating at the current station.
1. Keep the terminating trains there, but mark them in some obvious way. For example, instead of the station name put "(terminating)", maybe in italics.

Any preference?

**Comment by egoine** [comment]
Personally 1 would work for me but it wouldn’t work people interested in when a train will “arrive” (as opposed to when it leaves the station). I don’t know: maybe someone is interested to see when a train with a friend on will arrive? :)

Anyhow I think option 2 is safer at it keeps all the information available and also adds some information. 
I would fade it with a clear indication (as you suggested) so at a quick glance it’s obvious. 

In summary: option 2 :)

thanks!!

**Comment by egoine** [comment]
![495A914C-3B39-48FA-B2C7-766C6B8CFD17](https://user-images.githubusercontent.com/1479337/228641618-39c6867f-ad0b-4b69-a635-85ed03eaf62c.jpeg)


Mockup (doesn’t look good so probably not that useful! :)

**Comment by jamespfennell** [comment]
I wonder could we "fade out" out terminating trains, in the same way we fade out local stops that are not currently being served? E.g.: 155th st in this screenshot:
<img width="476" alt="image" src="https://github.com/jamespfennell/realtimerail.nyc-react/assets/12140490/f50daf1b-663d-4243-b532-061b086686e1">

**Comment by egoine** [comment]
That could work too. Tx

**Comment by jamespfennell** [comment]
I just implemented a very simple version of this in which trains that are terminating are marked as "(terminating train)" instead of having the destination stop. I want to do two follow-ups though:

- Have these trains faded out a little, as in my comment above.
- In a stop page that has one headsign of all terminating trains ("(terminating trains)", like 8av on the L), the terminating trains be at the bottom of the page. Right now they're at the top because the headsigns are presented in lexical order and "(terminating trains)" orders higher than any other headsign.

**Comment by jamespfennell** [comment]
Fixes are live on the site. Thanks for the suggestion!
